newIDE/app/src/UI/Checkbox.js

```diff
@@ -2,6 +2,16 @@
 import * as React from 'react';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Checkbox from '@material-ui/core/Checkbox';
+import { makeStyles } from '@material-ui/core/styles';
+
+// Reduce checkbox size to avoid overlapping with other checkboxes.
+const useStyles = makeStyles({
+  root: {
+    marginLeft: 9,
```

Comment:
Material UI was applying a padding of 9px and a margin of -9px, I'm not sure why... Maybe to have a large ripple effect around the checkbox.
Because we were reducing the size of the line, the checkbox were overlapping with top & bottom checkboxes

I'll double check everywhere